db/migrations/20230627115454_create_orders_table.sql

```diff
@@ -0,0 +1,16 @@
+-- migrate:up
+CREATE TABLE orders(
+  id INT NOT NULL AUTO_INCREMENT,
+  order_number INT NOT NULL,
```

Comment:
- order_number 데이터는 UUID를 사용해 주고 해당 데이터는 VARCHAR로 저장해 주세요.
```suggestion
  order_number VARCHAR(100) NOT NULL,
```

Reply:
UUID 참고 자료

https://kotlinworld.com/417#JavaScript%--UUID%--%EB%A-%-C%EB%--%A-%EA%B-%B-